android/src/main/java/fr/greweb/reactnativeviewshot/ViewShot.java

```diff
@@ -112,10 +118,20 @@ else if ("data-uri".equals(result)) {
     private void captureView (View view, OutputStream os) {
         int w = view.getWidth();
         int h = view.getHeight();
+
         if (w <= 0 || h <= 0) {
             throw new RuntimeException("Impossible to snapshot the view: view is invalid");
         }
-        Bitmap bitmap = Bitmap.createBitmap(view.getWidth(), view.getHeight(), Bitmap.Config.ARGB_8888);
+
+        //evaluate real height
+        if (this.isScrollView){
+            h=0;
+            ScrollView scrollView = (ScrollView)view;
+            for (int i = 0; i < scrollView.getChildCount(); i++) {
+                h += scrollView.getChildAt(i).getHeight();
```

Comment:
hi, i tested it, it can compute the realy height, when the one of the child have margin.